glusterd2/main.go

```diff
@@ -97,6 +100,15 @@ func main() {
 		log.WithError(err).Fatal("Could not add self details into etcd")
 	}
 
+	// Load the group option map into the store
```

Comment:
You can move this to a separate function (in volumes package) and call that function.